packages/react/src/components/Geo/types/maplibre-gl-geocoder.ts

```diff
@@ -0,0 +1,171 @@
+// `LocationSearchProps` is based upon the typing specified by maplibre-gl-geocoder:
+// https://github.com/maplibre/maplibre-gl-geocoder/blob/main/lib/index.js#L11-L66
+// TODO: formalize this type in the `maplibre-gl-geocoder` library
+export type LocationSearchProps = {
+  /**
+   * A bounding box given as an array in the format `[minX, minY, maxX, maxY]`.
```

Comment:
I wonder if we want to add `@description` to make it consist with the components, but it's ok to not add because we don't have the props table for connected-components